tfhe/benches/core_crypto/ks_bench.rs

```diff
@@ -3,13 +3,17 @@ use tfhe::core_crypto::prelude::*;
 use tfhe::keycache::NamedParam;
 use tfhe::shortint::prelude::*;
 
-fn criterion_bench(criterion: &mut Criterion) {
-    type Scalar = u64;
+fn keyswitch<Scalar: UnsignedTorus + CastInto<usize>>(criterion: &mut Criterion) {
+    let bench_name = "KS";
```

Comment:
Could you use the following name : `core_crypto::keyswitch` instead ?
The `core_crypto::` is an incoming task of mine to rename the benchmarks as the same model than integer ones.
This is done to ease parsing in Grafana and also for the internal data extractor.

Reply:
Isn't it missing in the pbs bench as well?

Reply:
Nope, in `pbs_bench.rs` everything is sent to Grafana. But in `pbs128_bench.rs` this is not the case, any idea why though @IceTDrinker ? Is it because it's some kind of experimental benchmarks ?

Reply:
@soonum I've changed all names, let me know if it's ok now

Reply:
Done

Reply:
yes pbs128 were experimental benchmarks though now that you mention it... we will need it for BC and stuff will add an issue